src/main/java/appeng/integration/modules/jei/GrinderRecipeCategory.java

```diff
@@ -88,12 +88,14 @@ public void setRecipe( IRecipeLayout recipeLayout, GrinderRecipeWrapper recipeWr
 		itemStacks.set( ingredients );
 	}
 
-	@Override public void addRecipeCategories( IRecipeCategory... recipeCategories )
+	@Override
+	public void addRecipeCategories( IRecipeCategory... recipeCategories )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)